### DISCUSSION TITLE
UX changes in 6.0 - feedback welcome

### DISCUSSION THREAD

**Discussion by janproch** [dbgate/dbgate#960] [General]
Did you try new UI, which comes with version 6.0 ?

There are significant changes in working with connections, databases and tables, do you enjoy these updates?

Any feedback is welcome. The perfect user experience is the main mission of DbGate

**Comment by gitter-me** [comment]
Yeah I tried and I'm really **disappointed about the removal of the query designer and the compare tab**.
Its a great tool and was recommended to me as an alternative to other tools. Its fast, powerful and works crossplatform.

But now it seems its moves to paid (only?) tool with a limited freemium / trial version.

What bothers me about this, are your words from your blog where you assured: 

> There are no plans to make **any limitations** of current desktop version. Because DbGate FREE and Premium shares the **same codebase**, DbGate FREE will profit from fixes and improvements made for its paid brother. ([ref](https://dbgate.org/development/2024/06/25/premium.html))


I understand that its complicated to run a business with open source, and I respect your decision if you need to make a living out of it. 

Don't get me wrong is still think its a cool software and I appreciate your work. 
But you said any feedback is welcome.

**Comment by janproch** [comment]
Thank you for your opinion. You are right, query designer and compare database tool were moved to premium version.

Compare database tool is tool for mainly professional users, there were lot of usability improvements of this tool, it was hard to mainain such tool in time, which we have reserved for community edition.

Moving query designer into premium was more complicated decision, but as I know all of DB tools, which implements query designer, has this feature in paid version. So this feature was exchanged for significant usability improvements, which were implemented in 6.0 and will continue in 6.1. Version 6.1 will contain fulltext quick search in DB model and suport of triggers, all these features will be available in community edition.

You probably noticed, that there is much more closed issues and missing features implemented after DbGate introduces the paid version, I am sure that todays DbGate Community is better than DbGate one year ago, event without query designer and compare DB tool.
